staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -98,6 +98,12 @@ const (
 
 	// LabelFailureDomainBetaRegion failure-domain region label
 	LabelFailureDomainBetaRegion = "failure-domain.beta.kubernetes.io/region"
+
+	// LabelTopologyZone
+	LabelTopologyZone = "topology.kubernetes.io/zone"
```

Comment:
Can we use v1. LabelTopologyZone instead of creating a new one? Similar to the FailureDomain label.

Reply:
Thanks for the suggestion, yes I was able to use v1 labels for those and made those changes